modules/data.remote/R/call_MODIS.R

```diff
@@ -111,11 +164,33 @@ call_MODIS <- function(outfolder = ".", start_date, end_date, lat, lon, size = 0
     output$lat <- round(output$lat, 4)
     output$lon <- round(output$lon, 4)
     
-    fname <- paste(product, "_", band, "output_", start_date, "_", end_date, "_", lat, "_", lon, ".csv", sep = "")
-    fname <- paste0(outfolder, "/", fname)
-    write.csv(output, fname)
-    return(output)}
-
+    if (QC_filter == T)
+    {
+      output$qc == as.character(output$qc)
+      for (i in 1:nrow(output))
+      {
+        convert = paste(binaryLogic::as.binary(as.integer(output$qc[i]), n = 8), collapse = "")
+        output$qc[i] = substr(convert, nchar(convert)-2, nchar(convert))
+      }
+      good = which(output$qc == "000" | output$qc == "001")
+      if (length(good) > 0 | !(is.null(good)))
+      {
+        output = output[good,]
+      } else {
+        print("All QC values are bad. No data to output with QC filter == TRUE.")
+      }
+    }
+    
+    if (!(outfolder == ""))
+    {
+      fname <- paste(product, "_", band, "_", siteID, "_output_", start_date, "_", end_date, ".csv", sep = "")
+      fname <- paste0(outfolder, "/", fname)
```

Comment:
See below comment about paths:
```suggestion
      fname <- file.path(outfolder, fname)
```

Reply:
Yes filepath which will deal with platform differences in paths